api/src/main/java/org/apache/iceberg/ManageSnapshots.java

```diff
@@ -163,26 +163,26 @@ default ManageSnapshots createBranch(String name) {
   ManageSnapshots replaceBranch(String name, long snapshotId);
 
   /**
-   * Replaces the branch with the given name to point to the source snapshot. The source branch will
-   * remain unchanged, the target branch will retain its retention properties.
+   * Replaces the source branch with the given name to point to the target snapshot. The target
+   * branch will remain unchanged, the source branch will retain its retention properties.
    *
-   * @param name Branch to replace
-   * @param source Source reference for the target to be replaced with
+   * @param source Branch to replace
+   * @param target Target reference for the target to be replaced with
```

Comment:
Should fix this java doc more:

should be 

 target Target reference for the source to be replaced with

Reply:
NIT: What are your thoughts on this declaration:  `fast_forward(String branchFrom, String branchTo)`

Reply:
Yeah@rdblue also suggested from and to. It sounded odd in combination with `replace` but I'll update it, maybe it's more natural when reading as opposed to how it sounds.

Reply:
I've updated to `from` and `to`. I think that still captures the difference (and the nice thing is it's more succinct and fall in line with some other code we have). Those prepositions still sounds a bit odd in my head (in my head replace should be combined with something like "with" but that would then not sound right with fast forward). combined with `replace` but I think that's minor. From and to still capture what each thing really is, and the desired end state.